src/Microsoft.Framework.PackageManager/SourceControl/ProcessUtilities.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal static class ProcessUtilities
+    {
+        public static bool ExecutableExists(string executableName)
+        {
+            string whereApp;
+            if (PlatformHelper.IsWindows)
+            {
+                whereApp = "where";
+            }
+            else
+            {
+                whereApp = "whereis";
```

Comment:
Is it guaranteed that `whereis` is always installed? Should we just start a subprocess and check whether it throws certain kind of exception?